docs/tutorials/style-your-graphs-in-memgraph-lab.md

```diff
@@ -82,9 +82,9 @@ Don't forget to click **Apply** to see your updated graph.
 
 ## Step 3 — Add images to the nodes
 
-Now that you have all the colors and borders in place, it is time to add images to the nodes. You will add images to the first and last node. As a source, you will use two images from Wikipedia. Each of the two nodes will have a different image. You will use a predicate to assign a value to a node with a specific node value.
+Now that all the colors and borders are just right, it's time to add images to the nodes. Let's add them to the first and last node using two different images from Wikipedia. You'll use a predicate to assign a value to a node with a specific node value.
```

Comment:
I think I would like to see an explanation of what a predicate is or how it generally looks?

Reply:
We have no documentation at the moment that mentions predicates. I think that explaining it here would be out of scope for this tutorial. One day when we will have docs we can link it from here.